src/EnergyPlus/ThermalComfort.cc

```diff
@@ -1214,11 +1212,32 @@ namespace ThermalComfort {
                 PPD_AD = (std::exp(-2.58 + 3.05 * AnkleAirVel - 1.06 * PMV) / (1 + std::exp(-2.58 + 3.05 * AnkleAirVel - 1.06 * PMV))) * 100.0;
 
             } else {
-                if (state.dataThermalComforts->FirstTimeAnkleDraftWarning) {
-                    ShowWarningError(state, "Ankle draft PPD calculations are only applicable for relative air velocity is below 0.2 m/s,");
-                    ShowContinueError(state,"and the subject’s metabolic rate and clothing level should be kept below 1.3 met and 0.7 clo.");
-                    ShowContinueError(state,"PPD at ankle draft will be set to -1.0 if if these conditions are not met.");
-                    state.dataThermalComforts->FirstTimeAnkleDraftWarning = false;
+                if (RelAirVel >= 0.2) {
+                    ShowRecurringWarningErrorAtEnd(state, "Relative air velocity is above 0.2 m/s in Ankle draft PPD calculations. PPD at ankle draft will be set to -1.0.",
```

Comment:
Thanks for adding these warnings. If I'm following correctly, all of these triggers (air velocity, metabolic rate, and clothing level) are user inputs, correct? So, maybe these warnings aren't so important, because this is likely the user's intent. How about wrapping all of these with `if (state.dataGlobal->DisplayExtraWarnings)` ? Sorry to keep changing my mind here.

Reply:
Just to be clear, my suggestion is to have one ` if (state.dataGlobal->DisplayExtraWarnings)` statement before line 1215 that skips all three of these warning checks.